pkg/crc/preflight/preflight_checks_common.go

```diff
@@ -47,7 +47,6 @@ var bundleExtractionCheck = Check{
 	check:            checkBundleExtracted,
 	fixDescription:   "Extracting bundle from the CRC executable",
 	fix:              fixBundleExtracted,
-	flags:            SetupOnly,
```

Comment:
When running during `start`, only `checkBundleExtracted` will be run, and `start` will fail if `checkBundleExtracted` fails, `fixBundleExtracted` won't be run during `start`. Not sure this is the behaviour we want.